packages/account-v2/tailwind.config.ts

```diff
@@ -10,9 +10,14 @@ export default {
                 solid: {
                     grey: {
                         '1': '#999999',
+                        '2': '#f2f3f4',
                         '5': '#d6dadb',
                         '6': '#d6d6d6',
                     },
+                    red: {
+                        '10': '#ff444f',
+                    },
+                    white: '#ffffff',
```

Comment:
We have `slate-50` as white, `coral-700` as red-10 from Quill. Please replace them

Reply:
Done.